public/data/PriyanshuIndra.json

```
@@ -0,0 +1,12 @@
+{
+    "name": "Priyanshu Indra",
+    "bio": "Student",
+    "avatar": "https://github.com/PriyanshuIndra.png",
+    "links": [
+        {
```

You can also add your GitHub profile. 😄
```suggestion
        {
            "name": "Follow me on GitHub",
            "url": "https://github.com/PriyanshuIndra",
            "icon": "github"
        },
        {
```

---

Yep, thanks !! Going to do that 😊